### ISSUE TITLE
about training！！！

### ISSUE THREAD

**Issue by wangxihao** [guoyww/AnimateDiff#202]
Hello, I would like to ask, how many video clips did you use in actual training? How long does it take to produce a preliminary animation effect after starting training with scratch? The effect of my training is very poor. I would like to refer to the author's training process. Thank you so much!

**Comment by tacit0428** [comment]
I have the same problem. I tried to train the motion module to fit a small dataset(about 3000 clips), but the loss didn't decrease. I trained 30 epochs, but the result looks the same. The training code and config are the same as the author provided docs.

**Comment by liutaocode** [comment]
same issue, stuck here for quite long time.

**Comment by supersonicMaclaurin** [comment]
same issue, have you sloved it ?
